pkgs/by-name/so/sops/package.nix

```diff
@@ -1,29 +1,24 @@
 {
   lib,
-  buildGo122Module,
+  buildGoModule,
   fetchFromGitHub,
   installShellFiles,
   versionCheckHook,
   nix-update-script,
 }:
 
-buildGo122Module rec {
+buildGoModule rec {
   pname = "sops";
-  version = "3.9.4";
+  version = "3.10.0";
 
   src = fetchFromGitHub {
     owner = "getsops";
     repo = pname;
     tag = "v${version}";
-    hash = "sha256-w2RMK1Fl/k8QXV68j0Kc6shtx4vQa07RCnpgHLM8c8Q=";
+    hash = "sha256-u5B2zjNg1Wnh8Ee240WEThhbJzxDcw/ib4u8oobTmh8=";
```

Comment:
```suggestion
    hash = "sha256-NOZvVL4b7+TVlB6iM4HJDa5PHOjvcN0BXDMOHmqg7lU=";
```

Reply:
Seems like upstream re-tagged the release. I was wondering why I got two release notifications for the same version :O